moveit_ros/perception/depth_image_octomap_updater/src/depth_image_octomap_updater.cpp

```diff
@@ -136,7 +136,7 @@ bool DepthImageOctomapUpdater::initialize(const rclcpp::Node::SharedPtr& node)
 
 void DepthImageOctomapUpdater::start()
 {
-  rmw_qos_profile_t custom_qos = rmw_qos_profile_system_default;
+  rmw_qos_profile_t custom_qos = rmw_qos_profile_sensor_data;
```

Comment:
`custom_qos` is only used once when creating the subscription, shouldn't we pass `rmw_qos_profile_sensor_data` directly?